src/tools/binaryen-shell.cpp

```diff
@@ -77,8 +77,8 @@ static void run_asserts(size_t* i, bool* checked, Module* wasm,
   std::unique_ptr<ShellExternalInterface> interface;
   std::unique_ptr<ModuleInstance> instance;
   if (wasm) {
-    interface = make_unique<ShellExternalInterface>();
-    instance = make_unique<ModuleInstance>(*wasm, interface.get());
+    interface = wasm::make_unique<ShellExternalInterface>();
```

Comment:
VS2015 says there's `std::make_unique` that is ambiguous with this, and fails to compile. The project defines its own `wasm::make_unique` in `src/support/utilities.h` that it conflicts with.